sdk/middlewares.go

```diff
@@ -4,14 +4,17 @@ import (
 	"encoding/json"
 	"net/http"
 
+	"github.com/urfave/negroni/v3"
```

Comment:
These middlewares were added in #59. We really need to wrap the `http.ResponseWriter` into a `negroni.ResponseWriter` so that we can use `rw.Status()` or `rw.Size()` in the middlewares. I'm adding a comment on the `Use` method.

`github.com/urfave/negroni/v3` is already used in go-handlers:

https://github.com/Scalingo/go-handlers/blob/1b60da7d270588a2085dbf957b10d7aad87f84d5/error_middleware.go#L12